db/models/aggregate_functions.py

```diff
@@ -223,7 +218,9 @@ def update_application_and_related_form(
     if form_name == "project-information":
         for question in question_json:
             for field in question["fields"]:
-                if (field["key"] == "KAgrBz") or (field["title"] == "Project name"):
+                if (field["key"] == "KAgrBz") or (
+                    field["title"] == "Project name"
```

Comment:
Will this part be affected by welsh translations?

Reply:
@adamdavies1 I think it would still work as long as we keep the keys consistent.